templates/zerver/integrations.html

```diff
@@ -969,6 +969,76 @@
 
     </div>
 
+    <div id="gcal" class="integration-instructions">
+
+        <p>Get Google Calendar reminders in Zulip! This is a great way to see your reminders directly in
+        your Zulip feed.</p>
+
```

Comment:
A step-by-step guide for that already exists [here](https://developers.google.com/google-apps/calendar/quickstart/python). Can I link this? Or should I rewrite and make it more general?